lib/galaxy/dependencies/pipfiles/default/pinned-requirements.txt

```diff
@@ -113,6 +114,7 @@ paste==3.5.0
 pastedeploy==2.1.1
 pastescript==3.2.0
 pbr==5.5.1; python_version >= '2.6'
+pillow==8.0.1
 prettytable==0.7.2
 protobuf==3.14.0
 prov==1.5.1
```

Comment:
Please add MGM dependencies in amp_requirements, as noted in the Galaxy upgrade documentation. Let's try not changing galaxy ones, unless there is a conflict - I'm not totally sure which ones overwrites the other, it might be galaxy overwrites ours. since ours is loaded during deploy, while galaxy one I believe is done during start. If we run into that scenario, we can evaluate a solution, maybe load ours in run.sh as well.